src/gui/dbsettings/DatabaseSettingsWidgetGeneral.ui

```diff
@@ -190,6 +190,58 @@
         </property>
        </widget>
       </item>
+      <item>
+       <layout class="QHBoxLayout" name="horizontalLayout_2">
+        <item>
+         <widget class="QCheckBox" name="autosaveDelayCheckBox">
+          <property name="toolTip">
+           <string>Autosave delay since last change</string>
+          </property>
+          <property name="accessibleName">
+           <string/>
```

Comment:
Should we provide a string here like you did on line 214?

Reply:
Forget it there, mb
Added at [71501a6](https://github.com/keepassxreboot/keepassxc/pull/9100/commits/71501a6ee6dff720ad31b6228c2dc1696ef2d197)